applications/nrf_desktop/src/modules/hids.c

```diff
@@ -501,6 +501,10 @@ static void notify_hids(const struct ble_peer_event *event)
 		__ASSERT_NO_MSG(cur_conn == event->id);
 		err = bt_gatt_hids_notify_disconnected(&hids_obj, event->id);
 
+		if (err) {
+			LOG_WRN("Connection was not secured");
```

Comment:
connection was or is

Reply:
I think it would be better to simply state "cannot notify hids" (the original message) as we don't know here the exact nature of an error.
btw - if it fails it can be troublesome; have you replicated any scenario where it fails?

Reply:
Connection was not secured before we disconnected.

I have taken a look into implementations and I noticed that its all about allocating and freeing memory for connection by module in the library.